src/main/java/domain/Car.java

```diff
@@ -3,13 +3,41 @@
 public class Car {
 
     private static final int MIN_VALUE = 4;
+    private static final int MAX_LENGTH = 5;
+    public static final String VALIDATION_MESSAGE_FORMAT = "길이는 %d를 초과할수없습니다.";
+
+    private static final String EX_MESSAGE = String.format(VALIDATION_MESSAGE_FORMAT, MAX_LENGTH);
+
 
     private int position;
+    private String name;
```

Comment:
학습 목적으로 name 변수를 래핑해볼까요?
그리고 이름 관련된 로직은 래핑한 객체로 모두 옮겨보세요.

```suggestion
    private CarName name;
```